modules/grafana/files/dashboards/email_alert_api_technical.json

```diff
@@ -883,54 +886,18 @@
           "nullPointMode": "connected",
           "percentage": false,
           "pointradius": 1,
-          "points": true,
+          "points": false,
           "renderer": "flot",
           "seriesOverrides": [],
           "spaceLength": 10,
           "span": 6,
-          "stack": true,
+          "stack": false,
           "steppedLine": false,
           "targets": [
             {
-              "bucketAggs": [
-                {
-                  "fake": true,
-                  "field": "route",
-                  "id": "4",
-                  "settings": {
-                    "min_doc_count": 1,
-                    "order": "desc",
-                    "orderBy": "_term",
-                    "size": "0"
-                  },
-                  "type": "terms"
-                },
-                {
-                  "fake": true,
-                  "field": "@timestamp",
-                  "id": "3",
-                  "settings": {
-                    "interval": "$Interval",
-                    "min_doc_count": 0,
-                    "trimEdges": 0
-                  },
-                  "type": "date_histogram"
-                }
-              ],
-              "dsType": "elasticsearch",
-              "hide": false,
-              "metrics": [
-                {
-                  "field": "view",
-                  "id": "1",
-                  "meta": {},
-                  "settings": {},
-                  "type": "count"
-                }
-              ],
-              "query": "application: email-alert-api AND status: [200 TO 299] AND route: status_updates*",
               "refId": "A",
-              "timeField": "@timestamp"
+              "target": "aliasByMetric(groupByNode(consolidateBy(summarize(stats_counts.govuk.app.email-alert-api.*.delivery_attempt.status.*, '$Interval', 'sum', false), 'sum'), 7, 'sum'))",
```

Comment:
Rather than delivery attempt statuses you should use the statuses that Notify send us, they are available at [`stats_counts.govuk.app.email-alert-api.*.status_update.status.*`](https://github.com/alphagov/email-alert-api/blob/8f7df7825fb5cefa5a41cbbcf272b85f3c1c426e/app/services/status_update_service.rb#L32)